src/openrct2-ui/windows/ShortcutKeys.cpp

```diff
@@ -87,127 +87,127 @@ struct ShortcutStringPair
+    { KeyboardShortcut::RideConstructionPreviousTrack,      STR_SHORTCUT_RIDE_CONSTRUCTION_PREVIOUS_TRACK },
+    { KeyboardShortcut::RideConstructionNextTrack,          STR_SHORTCUT_RIDE_CONSTRUCTION_NEXT_TRACK },
+    { KeyboardShortcut::RideConstructionBuildCurrent,       STR_SHORTCUT_RIDE_CONSTRUCTION_BUILD_CURRENT },
+    { KeyboardShortcut::RideConstructionDemolishCurrent,    STR_SHORTCUT_RIDE_CONSTRUCTION_DEMOLISH_CURRENT },
+
+    { KeyboardShortcut::Undefined,                             STR_NONE },
+
+    { KeyboardShortcut::ScrollMapUp,                         STR_SHORTCUT_SCROLL_MAP_UP },
+    { KeyboardShortcut::ScrollMapLeft,                       STR_SHORTCUT_SCROLL_MAP_LEFT },
+    { KeyboardShortcut::ScrollMapDown,                       STR_SHORTCUT_SCROLL_MAP_DOWN },
+    { KeyboardShortcut::ScrollMapRight,                      STR_SHORTCUT_SCROLL_MAP_RIGHT },
+
+    { KeyboardShortcut::Undefined,                             STR_NONE },
+
+    { KeyboardShortcut::WindowedModeToggle,                  STR_SHORTCUT_WINDOWED_MODE_TOGGLE },
+    { KeyboardShortcut::ScaleUp,                              STR_SHORTCUT_SCALE_UP },
+    { KeyboardShortcut::ScaleDown,                            STR_SHORTCUT_SCALE_DOWN },
+
+    { KeyboardShortcut::Undefined,                             STR_NONE },
+
+    { KeyboardShortcut::TileInspector,                        STR_SHORTCUT_OPEN_TILE_INSPECTOR },
+    { KeyboardShortcut::InsertCorruptElement,                STR_SHORTCUT_INSERT_CORRPUT_ELEMENT },
+    { KeyboardShortcut::CopyElement,                          STR_SHORTCUT_COPY_ELEMENT },
+    { KeyboardShortcut::PasteElement,                         STR_SHORTCUT_PASTE_ELEMENT },
+    { KeyboardShortcut::RemoveElement,                        STR_SHORTCUT_REMOVE_ELEMENT },
+    { KeyboardShortcut::MoveElementUp,                       STR_SHORTCUT_MOVE_ELEMENT_UP },
+    { KeyboardShortcut::MoveElementDown,                     STR_SHORTCUT_MOVE_ELEMENT_DOWN },
+    { KeyboardShortcut::IncreaseXCoord,                      STR_SHORTCUT_INCREASE_X_COORD },
+    { KeyboardShortcut::DecreaseXCoord,                      STR_SHORTCUT_DECREASE_X_COORD },
+    { KeyboardShortcut::IncreaseYCoord,                      STR_SHORTCUT_INCREASE_Y_COORD },
+    { KeyboardShortcut::DecreaseYCoord,                      STR_SHORTCUT_DECREASE_Y_COORD },
+    { KeyboardShortcut::IncreaseElementHeight,                  STR_SHORTCUT_INCREASE_ELEM_HEIGHT },
+    { KeyboardShortcut::DecreaseElementHeight,                  STR_SHORTCUT_DECREASE_ELEM_HEIGHT },
+
+    { KeyboardShortcut::Undefined,                             STR_NONE },
+
+    { KeyboardShortcut::AdvanceToNextTick,                  STR_ADVANCE_TO_NEXT_TICK },
+    { KeyboardShortcut::PaintOriginalToggle,                 STR_SHORTCUT_PAINT_ORIGINAL },
+    { KeyboardShortcut::DebugPaintToggle,                    STR_SHORTCUT_DEBUG_PAINT_TOGGLE },
```

Comment:
The second arguments were aligned before, so I think we shoult try to keep them so.